package.json

```diff
@@ -1,21 +1,22 @@
 {
   "name": "electrofishing",
-  "version": "2.4.9",
+  "version": "2.5.0-3",
+  "type": "module",
   "scripts": {
-    "build:stage": "export VITE_BUILD=stage && npm run build:sw && vite build",
+    "build:stage": "export VITE_BUILD=stage && pnpm run build:sw && vite build",
```

Comment:
i'm a little surprise you don't need a pnpm run after the &&

Reply:
Ya, I'm not sure how this is working TBH.